web/themes/joinup/templates/custom-form.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
```

Comment:
The template custom-form.html.twig renders content which is inside template core/themes/stable/templates/form/form.html.twig
`<form>`
content from custom-form.html.twig
`</form>`